examples/console/main.py

```diff
@@ -1,6 +1,25 @@
 """
+An example demonstrating a stand-alone "console" using JupyterLab.
```

Comment:
"using JupyterLab" seems misleading here.  Perhaps the 'Jupyterlab "console" in a stand-alone application'?